src/block_cache/block_cache.c

```diff
@@ -244,10 +245,20 @@ __blkcache_eviction_thread(void *arg)
             {
                 if (__blkcache_should_evict(session, blkcache_item, &reason)) {
                     TAILQ_REMOVE(&blkcache->hash[i], blkcache_item, hashq);
+
+                    data_size = blkcache_item->data_size;
```

Comment:
Does this belong better with the delta changes than the block manager changes?

Reply:
Possibly. It's a little hard to separate them out completely, because (1) the block cache does eventually need to know about deltas, and (2) I'm not going to rewrite it for this one PR when we're merging deltas soon anyway. I'm open to disagreements here.

Reply:
> when we're merging deltas soon anyway

Will the merge you are mentioning impact this code, @korteland ?